### PR TITLE
:bar_chart: Migrate covid datasets from covid-19-data

### PR DESCRIPTION
**Pull request by Marigold**
Migrate parts from [covid repository](https://github.com/owid/covid-19-data) that upsert data into MySQL into ETL. Covid scripts will take care of creating CSV files and ETL steps in here will put data from those CSVs into MySQL. Proof of concept for [Google mobility](https://github.com/owid/etl/pull/1236) has been already merged and works fine.

I ran chart revisions against staging server and everything looked good (except of maybe one chart, but I think it was an issue on my side)

It also includes recent country renames. 

## Remaining migration steps:

* [ ] Some datasets like [COVID-19 - Variants](https://owid.cloud/admin/datasets/5338) have prohibited redistribution, set it on new datasets in admin too (we can't set it from ETL)
* [ ] Run chart revisions on all COVID datasets
* [ ] Remove code that handles upserts to MySQL from covid-19-data repo https://github.com/owid/covid-19-data/pull/2807
* [ ] Archive old COVID datasets

### CODE REVIEW

File: etl/steps/data/garden/covid/latest/cases_and_deaths_who.meta.yml [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+
+      Our complete COVID-19 dataset is a collection of the COVID-19 data maintained
+      by <em>Our World in Data</em>. <strong>It
+      is updated daily</strong> and includes data on confirmed cases, deaths, hospitalizations,
+      and testing.
+
+
+      We have created a new description of all our data sources. You find it at our
+      GitHub repository <strong><a href="https://github.com/owid/covid-19-data/tree/master/public/data/">here</a></strong>.
+      There you can download all of our data.
```
Comment (lucasrodes) [inline etl/steps/data/garden/covid/latest/cases_and_deaths_who.meta.yml:17]: Avoid HTML

```
We have created a new description of all our data sources. You find it at our
      GitHub repository (https://github.com/owid/covid-19-data/tree/master/public/data/).
      There you can download all of our data.
```

File: etl/steps/data/garden/covid/latest/decoupling_metrics.meta.yml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    - Israel: <a href="https://datadashboard.health.gov.il/COVID-19/general">Government
+    of Israel</a>, automated via <a
+    href="https://github.com/dancarmoz/israel_moh_covid_dashboard_data/">dancarmoz
+    on GitHub</a>
+
+    - Spain: <a href="https://cnecovid.isciii.es/covid19/">National Center for Epidemiology</a>
+
+    - United States: <a href="https://covid.cdc.gov/covid-data-tracker/#trends_dailycases">US
+    CDC</a> and <a href="https://healthdata.gov/Hospital/COVID-19-Reported-Patient-Impact-and-Hospital-Capa/g62h-syeh">Department
+    of Health & Human Services</a>
```
Comment (lucasrodes) [inline etl/steps/data/garden/covid/latest/decoupling_metrics.meta.yml:43]: Avoid HTML

```
Germany: Robert Koch Institute (https://github.com/robert-koch-institut/)
```

Same for other countries

File: etl/steps/data/garden/covid/latest/ecdc.meta.yml
Comment (lucasrodes) [inline etl/steps/data/garden/covid/latest/ecdc.meta.yml:1]: Avoid HTML in dataset description field

File: etl/steps/data/garden/covid/latest/hospital_and_icu.meta.yml
Comment (lucasrodes) [inline etl/steps/data/garden/covid/latest/hospital_and_icu.meta.yml:1]: Avoid HTML in dataset description field

File: etl/steps/data/garden/covid/latest/john_hopkins_university.meta.yml
Comment (lucasrodes) [inline etl/steps/data/garden/covid/latest/john_hopkins_university.meta.yml:1]: Avoid HTML in dataset description field

File: etl/steps/data/garden/covid/latest/sequencing.meta.yml
Comment (lucasrodes) [inline etl/steps/data/garden/covid/latest/sequencing.meta.yml:1]: Avoid HTML in dataset description field

File: etl/steps/data/garden/covid/latest/sweden.meta.yml
Comment (lucasrodes) [inline etl/steps/data/garden/covid/latest/sweden.meta.yml:1]: Avoid HTML in dataset description field

File: etl/steps/data/garden/covid/latest/variants.meta.yml
Comment (lucasrodes) [inline etl/steps/data/garden/covid/latest/variants.meta.yml:1]: Avoid HTML in dataset description field.

I'd leave the one in lines 4-5 though for legal reasons.

**Review by lucasrodes — APPROVED** [review]
Thanks for doing this, @Marigold!

LGTM.

Just added minor comments on the metadata dataset's description. I think it should not use HTML tags. This issue was already present in the current scripts, but maybe we can remove these in this PR? However, this should not be a blocker for merging!

File: etl/steps/data/garden/covid/latest/cases_and_deaths_who.meta.yml [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+
+
+      The WHO licenses this data under CC BY-NC-SA 3.0 IGO. You can read more <a href="https://www.who.int/about/policies/publishing/copyright">here</a>.
```
[the remark this answers is not in the document]
Comment (Marigold) [reply etl/steps/data/garden/covid/latest/cases_and_deaths_who.meta.yml:20]: Why should we avoid HTML in dataset description? Does it apply also to variable / source description?

File: etl/steps/data/garden/covid/latest/cases_and_deaths_who.meta.yml [hunk 4]
```diff
@@ -0,0 +1,247 @@
+dataset:
+  title: COVID-19 Cases and deaths - WHO
```
[the remark this answers is not in the document]
Comment (Marigold) [reply etl/steps/data/garden/covid/latest/cases_and_deaths_who.meta.yml:2]: That makes sense, I've updated it. (By the way I haven't changed titles or descriptions of migrated datasets, they're identical to what we have right now. Perhaps we can clean it up & standardise in some future PR)

**Pull request merged**